projects/worker-azure-ai/package.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "azure-worker-ai",
+  "version": "0.0.1",
```

```suggestion
  "version": "0.1.0",
```
(1.0.0 would be fine too).

---

I guess use `1.0.0` consistent with [other example](https://github.com/adobe/asset-compute-example-workers/blob/master/projects/worker-metadata/package.json#L4) workers.